yarn.lock

```diff
@@ -1887,6 +1887,11 @@ anymatch@^3.0.3:
     normalize-path "^3.0.0"
```

Comment:
Please revert the changes on `yarn.lock`

Reply:
Reverted. I've read somewhere in the contribution guide that it needs to be included.